js/script.js

```diff
@@ -37,6 +37,15 @@ const updateTask = (task, newTitle) => {
   renderTasks();
 };
 
+const completeTask = (taskId) => {
+  const task = tasks.find((task) => task.id === taskId);
+  if (task && !task.done) {
+    task.done = true;
+    task.isEditing = false;
+    renderTasks();
+  }
+};
```

Comment:
Generally it's a good practice to reduce branches in code whenever we can.

```suggestion
const completeTask = (taskId) => {
  const task = tasks.find((task) => task.id === taskId);
  
  if (task === undefined){
    return
  }
  
  task.done = true;
  task.isEditing = false;
  
  renderTasks();
};
```

Reply:
Done